inlong-audit/audit-docker/audit-docker.sh

```diff
@@ -52,9 +52,16 @@ if [[ "${JDBC_URL}" =~ (.+):([0-9]+) ]]; then
   fi
 fi
 
-# start
-bash +x ${file_path}/bin/proxy-start.sh
-bash +x ${file_path}/bin/store-start.sh
+# start proxy
+cd ${file_path}/
```

Comment:
Double quote to prevent globbing and word splitting.
```suggestion
cd "${file_path}/"
```